### PR TITLE
10 shopinvader backend search tab

### PR DESCRIPTION
**Pull request by mourad-ehm**

### CODE REVIEW

File: shopinvader_search_engine/models/se_index.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+# Copyright 2013 Akretion (http://www.akretion.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+class SeIndex(models.Model):
+
+    _inherit = 'se.index'
+
+    is_valide = fields.Char(compute='_compute_is_valide')
```
Comment (bealdav) [inline shopinvader_search_engine/models/se_index.py]: Why is it a Char and not boolean ?

s/is_valide/is_valid

File: shopinvader_search_engine/models/se_index.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+    @api.depends('lang_id', 'model_id',)
+    def _compute_is_valide(self):
+        for rec in self:
+            active_id = self.env.context.get('shopinvader_backend_id')
+            active_id = self.env['shopinvader.backend'].browse(active_id)
```
Comment (hparfr) [inline shopinvader_search_engine/models/se_index.py:18]: should be put outside the for in self

File: shopinvader_search_engine/models/shopinvader_backend.py [hunk 3]
```diff
@@ -29,3 +39,22 @@ def clear_index(self):
...
+    def add_misssing_index(self):
+        self.ensure_one()
+        models = self._get_default_model()
+        index_obj = self.env['se.index']
+        ir_export_obj = self.env['ir.exports']
+        for model in models:
+            for lang in self.lang_ids:
+                if not self.index_ids.filtered(
+                        lambda i: i.lang_id == lang and i.model_id == model):
+                    ir_export = ir_export_obj.search([('resource', '=', model.model)])
```
Comment (lmignon) [inline shopinvader_search_engine/models/shopinvader_backend.py]: @mourad-ehm More than one 'ir.export' can be defined for same model.... `limit=1`

**Review by lmignon — CHANGES_REQUESTED** [review]

**Comment by lmignon** [comment]
conflicts resolved and code fixed

**Pull request merged**